error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ImmutableListRules.java

```diff
@@ -263,4 +263,84 @@ ImmutableList<T> after(T e1, T e2, T e3, T e4, T e5) {
       return ImmutableList.of(e1, e2, e3, e4, e5);
     }
   }
+
+  /**
+   * Prefer {@link Stream#of(Object[])} over constructing a list and then directly calling {@link
+   * ImmutableList#stream()} on it.
+   */
+  static final class StreamImmutableListOf1<T> {
+    @BeforeTemplate
+    Stream<T> before(T e1) {
+      return ImmutableList.of(e1).stream();
+    }
+
+    @AfterTemplate
+    Stream<T> after(T e1) {
+      return Stream.of(e1);
```

Comment:
Just like with the naming, we try to group based on their `@AfterTemplate`. As such I'll move these new rules to the `StreamRules` class.